generator/src/test/java/com/scottlogic/deg/generator/cucumber/Decimal.feature

```diff
@@ -0,0 +1,94 @@
+Feature: User can specify that a number is of type decimal and does not have any decimal places
```

Comment:
"does not have any decimal places"? Should this be "to a number of decimal places"? Or are you just saying that a decimal can be an integer?

Reply:
Oh I accidentally left it the same as for integer, good spot